edge-hub/src/Microsoft.Azure.Devices.Routing.Core/RouteFactory.cs

```diff
@@ -12,6 +12,12 @@ namespace Microsoft.Azure.Devices.Routing.Core
 
     public abstract class RouteFactory
     {
+        // Default route priority at half of uint.MaxValue,
+        // so that routes with no custom priority fall below
+        // everything else, and reserve room for routes that
+        // are below default priority.
+        public const uint DefaultPriority = uint.MaxValue / 2;
```

Comment:
UInt.MaxValue is 4294967295 and therefore default priority is 2147483647.  Should we round it and make it easier to remember, e.g. 2M?

Reply:
hmm...lemme think about it.

---
In reply to: [377447237](https://github.com/Azure/iotedge/pull/2469#discussion_r377447237) [](ancestors = 377447237)